### PR TITLE
Update docu

### PR DESCRIPTION
**Pull request by rwaffen**
add some real world examples

Fixes: #18

### CODE REVIEW

**Review by jay7x — APPROVED** [review]

File: examples/simple_setup/manifests/controller.pp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  Boolean $manage_kube_proxy                 = true,
+  Boolean $use_puppetdb                      = true,
+  Stdlib::HTTPUrl $master                    = 'https://kubernetes:6443',
+  K8s::Container_runtimes $container_manager = $profile::k8s::node::container_manager,
+  String[1] $etcd_version                    = '3.5.1',
+  String[1] $k8s_version                     = '1.26.1',
+  Enum['server'] $role                       = 'server',
+  K8s::CIDR $service_cidr                    = '10.20.0.0/20',
+  K8s::CIDR $pod_cidr                        = '10.20.16.0/20',
+) {
```
Comment (tuxmea) [inline examples/simple_setup/manifests/controller.pp:30]: Maybe specify if any of these parameters is a default parameter from the class.

File: examples/simple_setup/data/nodes/worker.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+classes:
+  - profile::k8s::worker
+
+### K8S::Node
+k8s::node::node_token: 'puppet.t0k3n' # generated by puppet, can be displayed with kubeadm token list
```
[the remark this answers is not in the document]
Comment (ananace) [reply examples/simple_setup/data/nodes/worker.yaml:6]: The token should be possible to specify on the controller side, but it seems I've forgotten to actually expose that parameter in `k8s::server::resources::bootstrap`. Will open a PR on that.

[the remark this answers is not in the document]
Comment (ananace) [reply examples/simple_setup/data/nodes/worker.yaml:6]: There we go, with #35 it should be possible to set the token secret using something like;
```yaml
# controller.yaml
k8s::server::resources::bootstrap::secret: 0123456789abcdef

# worker.yaml
k8s::node::node_token: puppet.0123456789abcdef
```

File: examples/simple_setup/manifests/controller.pp (hunk 1 above)
Comment (rwaffen) [reply examples/simple_setup/manifests/controller.pp:30]: i added a line to the readme about the structure and comments with the class default. is it this you wanted?

File: examples/simple_setup/data/nodes/controller.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### K8S::Server
+k8s::server::node_on_server: false
+
+k8s::server::etcd::generate_ca: true
+k8s::server::generate_ca: true
+k8s::server::manage_kubeadm: true
+
+### K8S::Server::Apiserver
+# Choose an interface which is for cluster communictaions.
```
Comment (tuxmea) [inline examples/simple_setup/data/nodes/controller.yaml:13]: Misspelling`communictaions` -> `communication`

**Review by tuxmea — APPROVED** [review]
Just one small typo

File: examples/simple_setup/data/nodes/controller.yaml (hunk 3 above)
Comment (rwaffen) [reply examples/simple_setup/data/nodes/controller.yaml:13]: oh thx. will fix.

**Pull request merged**